pennylane/devices/default_tensor.py

```diff
@@ -438,38 +455,45 @@ def _initial_quimb_circuit(
                 f"Unsupported gate contraction option: '{self._contract}' for '{self.method}' method. "
                 "Please refer to the documentation for the supported options."
             )
+        if psi0 is None:
+            psi0 = self._initial_mps(wires)
```

Comment:
This is supposed to be executed even if the user selects method=`mps`, right?

Reply:
Yes.